ui/address/AddressTxs.pw.tsx

```diff
@@ -47,6 +47,7 @@ base.describe('base view', () => {
     base.use({ viewport: configs.viewport.xl });
 
     base('', async() => {
+      base.slow();
```

Comment:
it is because there are a lot of hash-shortening calculations happening, and in CI it takes more time than locally for some reason. so I gotta increase the test timeout